modules/dnn/include/opencv2/dnn/all_layers.hpp

```diff
@@ -1127,7 +1127,6 @@ CV__DNN_INLINE_NS_BEGIN
     class CV_EXPORTS LayerNormLayer : public Layer
     {
     public:
-        bool hasBias;
```

Comment:
Breaks ABI check

Reply:
Yes, I know this. Actually I want to drop this variable. If we preserve ABI compatibility, we can keep it but not using it.

Reply:
Reverted this change. Lets keep it for compatiblity.